app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/data/database/GameGenieHistoryDao.kt

```diff
@@ -0,0 +1,27 @@
+package com.aptoide.android.aptoidegames.gamegenie.data.database
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.aptoide.android.aptoidegames.gamegenie.data.database.model.GameGenieHistoryEntity
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface GameGenieHistoryDao {
```

Comment:
most of these are only going to be used on the next ticket, should have been added there. Now it is already done so no problem